spark-shell/src/main/scala/org/apache/mahout/sparkbindings/shell/MahoutSparkILoop.scala

```diff
@@ -162,7 +163,7 @@ class MahoutSparkILoop extends SparkILoop {
          _ __ ___   __ _| |__   ___  _   _| |_
         | '_ ` _ \ / _` | '_ \ / _ \| | | | __|
         | | | | | | (_| | | | | (_) | |_| | |_
-        |_| |_| |_|\__,_|_| |_|\___/ \__,_|\__|  version 0.12.2
+        |_| |_| |_|\__,_|_| |_|\___/ \__,_|\__|  version 0.13.0
 
```

Comment:
I agree to keep.  The Zeppelin integration has a few of its own little issues, esp when it comes to serialization, and some other stuff.  The mahout-shell will always be way more 'bullet proof'.  I also like to sanity check myself, to figure out is an issues 1) Zeppelin's Fault, 2) Mahout's Fault 3) My Fault, by running code in the mahout-spark shell (after a zeppelin failure).